aws-rds-customdbengineversion/src/main/java/software/amazon/rds/customdbengineversion/ListHandler.java

```diff
@@ -13,6 +17,14 @@
 
 public class ListHandler extends BaseHandlerStd {
 
+    final static Collection<String> CUSTOM_ENGINES = ImmutableList.of(
+            "custom-oracle-ee",
+            "custom-oracle-ee-cdb",
+            "custom-sqlserver-ee",
+            "custom-sqlserver-se",
+            "custom-sqlserver-web"
```

Comment:
This list is likely to change over time. We need to have an action item to get these values dynamically.

Reply:
True, We are going to ask for a new API for this purpose from RDS service.